### PR TITLE
Xunit reporter

### LINKED ISSUE #10: xunit reporter

### PR DESCRIPTION
**Pull request by jkrall**
I banged out a simple xunit reporter, because I needed reports in Jenkins and the TAP reporter solution wasn't working for me.

Closes #10

### CODE REVIEW

File: lib/reporters/xunit.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  tag = content? (tag + content + '</' + name + end) : tag;
+  return tag;
+}
+
+function end(name) {
+  return '</' + name + '>';
+}
+
+function cdata(data) {
+  return '<![CDATA[' + utils.escape(data) + ']]>';
```
Comment (dasilvacontin) [inline lib/reporters/xunit.js]: Hi @jkrall! Any reason to wrap already escaped data inside a `CDATA` tag? Thanks much!

Comment (dasilvacontin) [inline lib/reporters/xunit.js]: ping @jkrall

Comment (jkrall) [inline lib/reporters/xunit.js]: Sorry... I can't remember.  This was over 4 years ago!

Comment (dasilvacontin) [inline lib/reporters/xunit.js]: No worries – thanks for replying though! :)

**Pull request merged**